mypyc/lib-rt/str_ops.c

```diff
@@ -201,4 +200,24 @@ Py_ssize_t CPyStr_Size_size_t(PyObject *str) {
         return PyUnicode_GET_LENGTH(str);
     }
     return -1;
-}
\ No newline at end of file
+}
+
+#define PyUnicode_UTF8(op)                              \
+    (assert(PyUnicode_Check(op)),                       \
+     assert(PyUnicode_IS_READY(op)),                    \
```

Comment:
This seems to crash the process in case the string object is not ready, which is not the right thing to do? Instead, you should probably use `PyUnicode_READY` and return NULL if it fails.